understanding/use-of-color.html

```diff
@@ -58,7 +81,7 @@ <h2>メリット</h2>
                <li>色覚異常の利用者が、色で伝えられている情報をその他の視覚的な手段で知覚できるようになる。</li>
                
                
-               <li>テキストしか表示しない、色数の制限された、又はモノクロのディスプレイを使用している人は、色に依存している情報を知覚できないかもしれない。</li>
+               <li>色数の制限されたモノクロのディスプレイを使用している人は、色に依存している情報を知覚できないかもしれない。</li>
```

Comment:
accessですから、「利用する」とでもしますか
```suggestion
               <li>色数の制限されたモノクロのディスプレイを使用している人は、色に依存している情報を利用できないことがある。</li>
```